dashboard/src/index.tsx

```diff
@@ -17,6 +17,14 @@ addAuthHeaders(axiosWithAuth);
 
 ReactDOM.render(<Root />, document.getElementById("root") as HTMLElement);
 
+// Allow to load custom styling.
+// The dashboard webserver will return this style file.
+const link = document.createElement("link");
+link.href = "/custom_style.css";
+link.type = "text/css";
+link.rel = "stylesheet";
+document.getElementsByTagName("head")[0].appendChild(link);
```

Comment:
If this file will be always present and you don't need to confirm anything with JS code, place the style link directly on the [index.html](https://github.com/kubeapps/kubeapps/blob/master/dashboard/public/index.html) template.